app/views/shared/_password_accordion.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= render AccordionComponent.new(class: 'margin-bottom-4') do |c| %>
   <% c.header { t('instructions.password.help_text_header') } %>
-  <%= simple_format(t('instructions.password.help_text')) %>
+  <% simple_format(t('instructions.password.help_text')) %>
```

Comment:
did the `=` get removed on purpose? is this supposed to render?

Reply:
Ah, that was a mistake. I was messing around with tests not working